lib/note-content-editor.tsx

```diff
@@ -150,8 +161,86 @@ class NoteContentEditor extends Component<Props> {
         }
       }, 400);
     }
+
+    if (this.editor && prevProps.searchQuery !== this.props.searchQuery) {
```

Comment:
we also need to gate this on `this.state.editor === 'full'` otherwise we'll be trying to update the decorators for a plain `<div>`

Reply:
Done